src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts

```diff
@@ -12,28 +12,27 @@ export class ClickableButtonRendererComponent implements ICellRendererAngularCom
     label: string;
     isIcon = false;
     icon: string;
+    color: string;
     private params: any;
 
-    agInit(params: any): void {
-        this.params = params;
+    setRendererParams(params) {
```

Comment:
Change the name of this method to express its usage inside this component.
Also, make this method private.